util/helpers.py

```diff
@@ -28,39 +30,31 @@ def check_rclone_exists():
     return ret
 
 
-def convert_bytes_to_best_unit(bytes_value):
-    bytes_value = float(bytes_value)
+# Inspired from https://stackoverflow.com/a/14822210
+def convert_bytes_to_best_unit(bytes):
+   if bytes == 0:
+       return "0 Bytes"
+   sizes = ("Bytes", "kB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB")
+   i = int(math.floor(math.log(bytes, 1024)))
+   s = round(bytes / math.pow(1024, i), 2)
+   return str(s) + sizes[i]
 
-    value_tmp = bytes_value * 1e-15
-    if value_tmp >= 1:
-        return str(round(value_tmp, 1)) + "PB"
-
-    value_tmp = bytes_value * 1e-12
-    if value_tmp >= 1:
-        return str(round(value_tmp, 1)) + "TB"
-
-    value_tmp = bytes_value * 1e-9
-    if value_tmp >= 1:
-        return str(round(value_tmp, 1)) + "GB"
-
-    value_tmp = bytes_value * 1e-6
-    if value_tmp >= 1:
-        return str(round(value_tmp, 1)) + "MB"
-
-    value_tmp = bytes_value * 1e-3
-    if value_tmp >= 1:
-        return str(round(value_tmp, 1)) + "kB"
-
-    return str(bytes_value) + "B"
 
 # Calculate path size in bytes using RClone
-def calculate_path_size(path, config_file):
-    response = subprocess.check_output('rclone --config {} size \"{}\"'.format(config_file, path), shell=True, stderr=subprocess.DEVNULL)
-    response_processed = response.decode('utf-8').replace('\0', '')
-    response_bytes = response_processed.split('(')[1]
-    response_bytes = response_bytes.replace('Bytes)', '').strip()
-
-    return response_bytes
+def calculate_path_size(config_file, src_label, dst_label):
+    cmd = ['rclone', '--config', config_file, 'copy', '--dry-run', src_label, dst_label]
```

Comment:
Can you change this so that instead of hardcoded `copy` it uses the supplied method through args (--sync, --copy, --move) etc. to ensure accuracy.